ocaml/parsing/pprintast.ml

```diff
@@ -687,9 +694,15 @@ and sugar_expr ctxt f e =
     end
   | _ -> false
 
-and expression ctxt f x =
+(* Jane Syntax defies the property "if an expression has attributes,
```

Comment:
I don't quite follow the plan here. It looks like `jane_syntax_expr` looks to see if there are attributes after peeling off the jane-syntax attributes, so isn't that sufficient?